docs/releases/1.9.txt

```diff
@@ -301,6 +301,10 @@ Forms
 
 * You can now :ref:`specify keyword arguments <custom-formset-form-kwargs>`
   that you want to pass to the constructor of forms in a formset.
+* :class:`~django.forms.CharField` now accepts a
```

Comment:
blank line between items in release notes (auto-merge probably stripped them out)